.github/actions/test-dotnet/Invoke-SolutionTests.ps1

```diff
@@ -14,12 +14,12 @@ $ConnectionStringSuffix = ";MultipleActiveResultSets=True;Connection Timeout=60;
 if ($Env:RUNNER_OS -eq "Windows")
 {
     $Env:Lombiq_Tests_UI__SqlServerDatabaseConfiguration__ConnectionStringTemplate =
-        "Server=.\SQLEXPRESS;Database=LombiqUITestingToolbox_{{id}};Integrated Security=True" + $ConnectionStringSuffix
+    "Server=.\SQLEXPRESS;Database=LombiqUITestingToolbox_{{id}};Integrated Security=True" + $ConnectionStringSuffix
```

Comment:
Here and on line 22 the original indentation was good.

Reply:
Autoformatting again. :/